samples/subsys/tracing/sample.yaml

```diff
@@ -23,7 +23,7 @@ tests:
       - CONFIG_MP_NUM_CPUS=1
       - CONFIG_OPENOCD_SUPPORT=y
       - CONFIG_SMP=n
-    arch_exclude: posix xtensa
+    arch_exclude: posix xtensa sparc
```

Comment:
I'd appreciate filing an Enhancement ticket to eventually get tracing working.

Reply:
Agreed about filing enhancement ticket to add tracing. Would it be an appropriate way forward to file that ticket after (if) this pull request is accepted?

The `arch_exclude` in `samples/subsys/tracing/sample.yaml` was specific to the `tracing.osawareness.openocd` sample. It was possible to eliminated the `arch_exclude` by providing the corresponding support in `subsys/debug/openocd.c` instead. That update is also covered by the 2020-11-12 force-push.

Reply:
just open an Enhancement on GitHub summarizing the situation.

Reply:
OK!